.gitignore

```diff
@@ -5,3 +5,4 @@ data/contributors/contributor.yaml
 deploy.sh
 linux/
 **gource**
```

Comment:
Should this be **source**

Reply:
It is spelled as intended here. Gource is a tool to visualize the lifecycle of a repository and all of the commits that were made.